Tests/Svg.UnitTests/ImageComparisonTest.cs

```diff
@@ -115,14 +123,16 @@ private void CompareSvgImageWithReferenceImpl(string baseName, string svgPath, s
         public void RecordDiffForAllSvgImagesWithReference()
         {
 #if NETSTANDARD || NETCOREAPP
-            var basePath = Path.Combine(ImageTestDataSource.SuiteTestsFolder, "W3CTestSuite");
+            var testsRoot = ImageTestDataSource.SuiteTestsFolder;
 #else
-            var basePath = Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(TestContext.TestDirectory))); //TODO: Tthe get dir name was parsed from the testparams -> Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(TestContext.TestRunDirectory)));
-            basePath = Path.Combine(Path.Combine(basePath, "Tests"), "W3CTestSuite");
+            var testsRoot = Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(TestContext.TestDirectory))); //TODO: Tthe get dir name was parsed from the testparams -> Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(TestContext.TestRunDirectory)));

```

Comment:
I'm not sure what this TODO even means - maybe remove it unless you get it

Reply:
Observations
- The target method `RecordDiffForAllSvgImagesWithReference` is not marked as `[Test]` so it is not used: i.e. it is enabled.
- `RecordDiffForAllSvgImagesWithReference` is not a unit test function, only used to observe the percentage of difference in image comparison.
- The file referenced `..\..\..\..\Tests\Svg.UnitTests\all.csv` does not exist.
- It is uses a property `public TestContext TestContext { get; set; }` which is not initialized, so I think the **TODO** is a reminder that you will get a null exception.

Changes
- Changed the `..\Tests\Svg.UnitTests\all.csv` to `..\Tests\Svg.UnitTests\AllTests.csv`, which exists.
- Initialized the property `public TestContext TestContext { get; set; }`  and actually used it.

Tests
- I enabled the `RecordDiffForAllSvgImagesWithReference` and tested it on the support frameworks.
- Disabled it after the tests; normal condition restored 😄